Core/TransactionHandler.php

```diff
@@ -0,0 +1,293 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Core;
+
+use \Exception;
+use \OxidEsales\Eshop\Application\Model\Order;
+use OxidEsales\Eshop\Core\DatabaseProvider;
+use \OxidEsales\Eshop\Core\Registry;
+use \OxidEsales\Eshop\Core\Field;
+
+use Wirecard\Oxid\Model\PaymentMethod;
+use \Wirecard\PaymentSdk\Entity\Amount;
+use \Wirecard\PaymentSdk\Response\SuccessResponse;
+use \Wirecard\PaymentSdk\Response\FailureResponse;
+use \Wirecard\PaymentSdk\Response\Response;
+use \Wirecard\PaymentSdk\BackendService;
+
+use \Wirecard\Oxid\Model\Transaction;
+
+/**
+ * Class TransactionHandler
+ * @package Wirecard\Oxid\Core
+ */
```

Comment:
I think we always use mixin and not package
also since is missing in the whole file